GOVERNANCE.md

```diff
@@ -28,20 +26,27 @@ No more than one-fourth of the CC members may be affiliated with the same employ
 
 The CC members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
 
+The CC may, at its discretion, invite any number of non-voting observers to participate in the public portion of CC discussions and meetings.
+
 The CC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the CC). The meeting shall be directed by the Individual Membership Directors. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
 
+CC members are expected to regularly participate in CC activities.
+
+In the case where an individual CC member -- within any three month period -- attends fewer than 25% of the regularly scheduled meetings, does not participate in CC discussions, and does not participate in CC votes, the member shall be automatically removed from the CC. The member may be invited to continue attending CC meetings as an observer.
+
 ## Section 5. Responsibilities of the CC.
 
 Subject to such policies as may be set by the Board, the CC is responsible for all cultural development and outreach within the Node.js Foundation, including:
 
 - Outreach to community organizations
 - Documentation of community organizations
 - Cultural direction of Node.js Foundation
-- Overseeing Inclusivity WG
+- Overseeing the Inclusivity WG
+- Overseeing education initiatives for the Node.js Foundation
 - Project governance and process (including this policy)
 - Recommendations for building and developing community projects that align with needs of Node.js
 - Mediating cultural conflicts between Foundation and/or community projects
-- Facilitation with external open source projects, select consortiums and other outside groups.
+- Facilitation with external open source projects, select consortiums and other outside groups
```

Comment:
Quick note: it's not particularly clear what "facilitation" means in this context.